### PR TITLE
Refactor tests

### PR DESCRIPTION
**Pull request by mrdrogdrog**
### Description
This PR refactors the test file and moves it into the src directory to use eslint.

### Steps

- [x] Added changes
- [x] I read the [contribution documentation](https://github.com/hedgedoc/markdown-it-better-task-lists/blob/main/CONTRIBUTING.md) and signed-off my commits to accept the DCO.

### CODE REVIEW

File: src/__test/fixtures/load-fixtures.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+}
+
+/**
+ * Loads a fixture file
+ *
+ * @param fileName The name of the fixture file
+ * @return the loaded fixture
+ */
+export function loadFixture(fileName: FixtureFiles): Fixture {
+  const mdParser = new MarkdownIt().use(taskLists)
```
Comment (DerMolly) [inline src/__test/fixtures/load-fixtures.ts:49]: Is it a good idea to create a new markdownit for each test?

**Review by DerMolly — COMMENTED** [review]
Also I'm unsure of the __test directory? This seems unnecessary and in contrast to the rest of the repos…

File: src/__test/fixtures/load-fixtures.ts (hunk 1 above)
Comment (mrdrogdrog) [reply src/__test/fixtures/load-fixtures.ts:49]: It's not for every test. It's for every fixture. So per complete jest run it happens 4 times. imho that's okay.

**Comment by mrdrogdrog** [comment]
@DerMolly About the test directory: It's totally common in jest to have a  `__test__` directory that contains test related files

**Review by DerMolly — APPROVED** [review]

**Pull request merged**